lib/base/tlsstream.cpp

```diff
@@ -315,14 +317,13 @@ void TlsStream::Handshake()
 	m_CurrentAction = TlsActionHandshake;
 	ChangeEvents(POLLOUT);
 
-	boost::system_time const timeout = boost::get_system_time() + boost::posix_time::seconds(TLS_TIMEOUT_SECONDS);
+	boost::system_time const timeout = boost::get_system_time() + boost::posix_time::milliseconds(long(Configuration::TlsHandshakeTimeout * 1000));
```

Comment:
Why did you change this from seconds to milliseconds? I don't see the point.

Reply:
Because the timeout can be configured as double, and otherwise you couldn't pass 1.555 for example.

Reply:
Seems legit 👍